### PR TITLE
BOM-1579 : Update test matrix

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA : https://openedx.atlassian.net/browse/BOM-1579

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,10 @@
 [tox]
-envlist = csslint,eslint,pycodestyle,pylint,py35-django{111,20,21,22}
+envlist = csslint,eslint,pycodestyle,pylint,py35-django{22,30}
```
Comment (mraarif) [inline tox.ini:2]: shouldn't we add python 3.8 here as well?

Comment (mzulqarnain1) [reply tox.ini]: You are right. I've added it.

File: .travis.yml [hunk 2]
```diff
@@ -26,7 +24,7 @@ install:
   - "make requirements"
   - "pip install coveralls"
 script:
-  - make test
+  - tox -e $TOXENV
```
Comment (morenol) [inline .travis.yml:27]: I think that this can be changed with just `tox`

File: tox.ini [hunk 3]
```diff
@@ -1,8 +1,10 @@
 [tox]
-envlist = csslint,eslint,pycodestyle,pylint,py35-django{111,20,21,22}
+envlist = csslint,eslint,pycodestyle,pylint,py{35,38}-django{22,30}
```
Comment (morenol) [inline tox.ini:2]: Django 3.0 with Python3.5 is an invalid combination

File: .travis.yml (hunk 2 above)
Comment (mzulqarnain1) [reply .travis.yml]: done/

**Comment by mzulqarnain1** [comment]
> LGTM. Could you create a new release?

@jmbowman Do we need to make a release in these forks since we install these using their latest commit hash in the platform configuration.

**Comment by jmbowman** [comment]
No, the ones like this that haven't been released to PyPI don't need to be right now.  We'll probably have a separate project later to get all these released to PyPI so we can install fewer things directly from GitHub.

**Pull request merged**